packages/react/src/alert/Alert.css.ts

```diff
@@ -71,14 +47,23 @@ export const icon = recipe({
       mt: "4",
       w: "auto",
     },
+    style({
+      color: iconColorVar,
+    }),
   ],
+
   variants: {
     colorScheme: {
       danger: style({
         vars: {
           [iconColorVar]: theme.colors["bg.error.solid.hover"],
         },
       }),
+      information: style({
+        vars: {
+          [iconColorVar]: theme.colors["brand.600"],
+        },
+      }),
```

Comment:
We can use sprinkles for each colorScheme, instead of keeping the value in `var`